codalab/lib/bundle_cli.py

```diff
@@ -1444,6 +1444,13 @@ def copy_bundle(
 
         print("Copying %s..." % source_desc, file=self.stdout)
 
+        # If bundle contents don't exist, finish after just copying metadata
+        try:
```

Comment:
Bundles that have been `rm -d` just have their `data_hash` missing, so I think it's semantically more accurate (and faster) to just check that (look in the `metadata`).